src/main/java/com/money_account_service/configuration/AppConfig.java

```diff
@@ -56,6 +57,11 @@ public OkHttpClient okHttpClient() throws NoSuchAlgorithmException, KeyManagemen
         return builder.build();
     }
 
+    @Bean
+    public DateTimeProvider localDateTime() {
```

Comment:
I'm not sure if you need custom class for this. I understand its for testing purpose to handle methods like `Instant.now()`. In many projects I used `Clock clock`, which has appropiate methods. https://www.baeldung.com/java-override-system-time

Reply:
`clock.instant()` https://www.baeldung.com/java-clock